pkg/kubelet/lifecycle/predicate.go

```diff
@@ -128,7 +128,7 @@ func (w *predicateAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 		switch re := r.(type) {
 		case *predicates.PredicateFailureError:
 			reason = re.PredicateName
```

Comment:
Why not combine reason and message together?

Reply:
done, thx

Reply:
I mean reason can be considered a failure cause type, and the message is the details of the reason for the failure.
`
PodAdmitResult{			
			Admit:   fit,				
			Reason:  reason,	
			Message: message,
		}
`
will combine them.